src/status_im/core.cljs

```diff
@@ -8,7 +8,10 @@
             [status-im.react-native.js-dependencies :as js-dependencies]
             [goog.object :as object]))
 
-(when js/goog.DEBUG
+(defonce debug?
```

Comment:
What does this provide us?

Reply:
A way to know which version is running. But it only shows cljs build, not android build.

Reply:
As opposed to using `js/goog.DEBUG` directly?

Reply:
just convenience :) Or it is not necessary?

Reply:
I'd rather stick with the default convention

Reply:
ok, I remove it then.